src/engine_redraw.c

```diff
@@ -518,6 +518,23 @@ void redraw_creature_view(void)
     message_draw();
     gui_draw_all_boxes();
     draw_tooltip();
+    long x = 148 * units_per_pixel / 16;
```

Comment:
and here too,... those macros I pointed to on the chinese scaling PR

Reply:
What Chinese scaling PR? I can't see it.

Reply:
Oh, you mean the message box one?